client/src/routes/user/ListPage.tsx

```diff
@@ -221,6 +267,10 @@ function isMotstandenMail(email: string): boolean {
     return email.trim().toLowerCase().endsWith("@motstanden.no")
 }
 
+function isActive(status: string): boolean {
+    return status == UserStatus.Active
+}
+
```

Comment:
Foreslår å gjøre dette inline. Unødvendig å ha en egen funksjon for så enkel logikk